### PR TITLE
Update: add mpc paper

### PR DESCRIPTION
**Pull request by BeStrongok**
**Type of change**

- [x] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

> Please include a summary of the change and the relevant motivation.
Add a new **Overdrive LowGear 2.0** paper:
This paper presents a new offline phase based on linear homomorphic encryption that reduces the bandwidth and number of communications rounds compared to the state-of-the-art MPC protocol for a small number of parties, as well as matrix triples generation, which i think it could be utilized in low bandwidth scenarios.
Ask @Jamie-Cui for reviewing :)

### CODE REVIEW

**Comment by BeStrongok** [comment]
Hi, @Jamie-Cui

File: docs/awesome-pets/papers/tools/mpc.md [hunk 1]
```diff
@@ -154,6 +154,12 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
   *Elette Boyle, Geoffroy Couteau, Niv Gilboa, Yuval Ishai, Lisa Kohl, Nicolas Resch, Peter Scholl*  
   Crypto 2022, [eprint](https://eprint.iacr.org/2022/1014), BCG+22
 
+### Others
+
+- Overdrive LowGear 2.0: Reduced-Bandwidth MPC without Sacrifice
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/tools/mpc.md:159]: [style] Can you add two spaces here? Thanks :P

File: docs/awesome-pets/papers/tools/mpc.md [hunk 2]
```diff
@@ -154,6 +154,12 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
...
+  *Sebastian Hasler, Toomas Krips, Ralf Küsters, Pascal Reisert, Marc Rivinius*
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/tools/mpc.md:160]: Same for this line

**Review by Jamie-Cui — CHANGES_REQUESTED** [review]

File: docs/awesome-pets/papers/tools/mpc.md (hunk 1 above)
Comment (Jamie-Cui) [reply docs/awesome-pets/papers/tools/mpc.md:159]: ** At the end of this line

File: docs/awesome-pets/papers/tools/mpc.md (hunk 2 above)
Comment (BeStrongok) [reply docs/awesome-pets/papers/tools/mpc.md]: Yes, i have fixed the style issue.

**Review by Jamie-Cui — APPROVED** [review]
LGTM

**Pull request merged**